src/main/java/eu/enmeshed/model/relationships/Relationship.java

```diff
@@ -17,27 +17,19 @@
 @Setter
 @Builder
 public class Relationship implements WebhookData {
-
-  private String id;
-
-  private RelationshipTemplate template;
-
-  private Status status;
-
-  private String peer;
-
-  private IdentityInfo peerIdentity;
-
-  private List<RelationshipChange> changes;
-
-  public enum Status {
-    @JsonProperty("Pending")
-    PENDING,
-
-    @JsonProperty("Active")
-    ACTIVE,
-
-    @JsonProperty("Rejected")
-    REJECTED
-  }
+    private String id;
+    private RelationshipTemplate template;
+    private RelationshipStaus status;
+    private String peer;
+    private IdentityInfo peerIdentity;
+    private RelationshipCreationContent creationContent;
+    private List<RelationshipAuditLog> auditLog;
+    private PeerDeletionInfo peerDeletionInfo;
+
+    public enum PeerDeletionInfo {
+        @JsonProperty("ToBeDeleted")
+        TO_BE_DELETED,
+        @JsonProperty("Deleted")
+        DELETED
+    }
```

Comment:
PeerDeletionInfo is an object containing a status enum:

```
export enum PeerDeletionStatus {
    ToBeDeleted = "ToBeDeleted",
    Deleted = "Deleted"
}

export interface PeerDeletionInfo {
    deletionStatus: PeerDeletionStatus;
}
```

Reply:
done